config/stylelint/sassMsgFormatter.js

```diff
@@ -0,0 +1,119 @@
+const chalk = require('chalk');
+
+/**
+ * @constant
+ */
+const TITLE = chalk.bgYellow;
+const ERROR = chalk.bold.red;
+const WARNING = chalk.yellow;
+const URL = chalk.underline.cyan;
+const NUMBER = chalk.dim;
+const RULE = chalk.bgWhite.black;
+
+/**
+ * returns the boolean representing whether a file has any errors for filtering purposes
+ * @param {Object} result - the object representing the result of a linting session
+ * @returns {Boolean} returns true or false depending on whether the file has errors or not
+ */
+function filterForErrors(result) {
+  return result.errored;
+}
```

Comment:
Why the duplication of functions defined in `config/stylelint/sassMsgFormatterFunctions.js`? Should these functions be imported from the functions file? Or could the functions file be deleted and the test import exported functions from this file?

Reply:
Resolved, looks solid!